vcpkg/users/port-overlays.md

```diff
@@ -0,0 +1,80 @@
+---
+title: Port overlays
+description: This article describes overlay ports in vcpkg. Overlay ports are used to force a specific port definition to be used by vcpkg during the package installation process.
+author: data-queue
+ms.author: danshaw2
+ms.date: 9/20/2023
+ms.prod: vcpkg
+ms.topic: conceptual
+---
+
+# Overlay ports
+
+## Overview
+An overlay port can act as a drop-in replacement for an existing port. An overlay port can also add a new port that is otherwise not available in a [registry](../maintainers/registries.md). While [resolving package names](registries.md#package-name-resolution), overlay ports take priority, forcing overlay ports to be selected.
+
+Naming conflicts can arise when multiple overlays introduce the same port. For instance, overlay directories `[dirA, dirB]` may both introduce a port with the same name: `dirA/zlib` and `dirB/zlib`. vcpkg resolves `zlib` in order of which directory is specified first. In this example, `dirA/zlib` is selected as the `zlib` port. If overlay port options are specified in multiple places, overlay resolution priority follows first from the command line, the manifest, and finally the environment.
+
+## Configuration Syntax
```

Comment:
```suggestion
## Using an overlay port
```
?

Do we have docs for creating an overlay port? That way, we can describe it and link to a tutorial for creating an overlay port. I ask because the crux of this article is describing how to use an overlay port.

Reply:
Yes, we have an article "Use a locally modified dependency" (or some similar title) which is intended to be a 1-2 combo of applying patches and using overlays tutorials.

Reply:
fixed